docs/imexam/dependencies.rst

```diff
@@ -2,18 +2,18 @@
 Software Dependencies
 *********************
 
-*   Astropy (for some analysis functions)
-*   photutils (for photometry)
+Required
+--------
+
+*   Python 2.7 or 3.3+ (Python 2.6 and 3.2 are not supported)
+*   numpy >= 1.6.0
+*   Astropy >= 0.3 (for some analysis functions)
 *   matplotlib (for plotting)
 *   DS9 (for image display) 
 
-astropy >= 0.3
-
-python >= 2.7
-
-numpy >= 1.6.0
+Optional
+--------
 
-photutils > 0.0.dev
-    This must be installed to enable the photometry options for imexam() but it is not required
+*   scipy (optional, for curve fitting)
+*   photutils development version (optional, for photometry options in imexam())
```

Comment:
I think I've fixed up all the requirements with the current code....I have a large pull request that I'll merge today which has new functionality, but should also work with the current release of photutils and astropy 1.